src/prefect/orion/database/migrations/versions/postgresql/2022_11_24_143620_f7587d6c5776_add_worker_tables.py

```diff
@@ -0,0 +1,239 @@
+"""Add worker tables
+
+Revision ID: f7587d6c5776
+Revises: 5d526270ddb4
+Create Date: 2022-11-24 14:36:20.350834
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+import prefect
+
+# revision identifiers, used by Alembic.
+revision = "f7587d6c5776"
+down_revision = "5e4f924ff96c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "worker_pool",
+        sa.Column(
+            "id",
+            prefect.orion.utilities.database.UUID(),
+            server_default=sa.text("(GEN_RANDOM_UUID())"),
+            nullable=False,
+        ),
+        sa.Column(
+            "created",
+            prefect.orion.utilities.database.Timestamp(timezone=True),
+            server_default=sa.text("CURRENT_TIMESTAMP"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated",
+            prefect.orion.utilities.database.Timestamp(timezone=True),
+            server_default=sa.text("CURRENT_TIMESTAMP"),
+            nullable=False,
+        ),
+        sa.Column("name", sa.String(), nullable=False),
+        sa.Column("description", sa.String(), nullable=True),
+        sa.Column("type", sa.String(), nullable=True),
```

Comment:
I believe we'll need a column for the base job definition (which implicitly will define the `type`)